RELEASE_NOTES.md

```diff
@@ -19,7 +19,17 @@ Notable changes include:
 
   * Bug fixes/improvements:
 
-Version 2022.10.4 -- Release date 2022-14-01
+Version 2022.10.5 -- Release date 2023-02-xx
```

Comment:
Does this date need to be updated?

Reply:
Yeah, I saw that and have a local fix I need to push.